tests/t001-ceed.c

```diff
@@ -0,0 +1,19 @@
+/// @file
+/// Test creation and destruction of a CEED object
+/// \test Test creation and destruction of a CEED object
+#include <ceed.h>
+#include <limits.h>
+
+int main(int argc, char **argv) {
+  Ceed ceed;
+  CeedMemType type = INT_MAX;
```

Comment:
I guess I don't see why this is meaningfully distinct from the test below, but the idea with numbers was to make the sorted list have some meaning and be easy to navigate. If we have multiple tests with the same number, we lose that.

Reply:
Stray file, removed